frontend/resourceadm/components/MigrationPanel/MigrationPanel.tsx

```diff
@@ -67,6 +70,44 @@ export const MigrationPanel = ({
     setServiceExpiredWarningModalRef.current?.close();
   };
 
+  const getErrorMessage = (): React.ReactNode => {
```

Comment:
Maybe we should refactor this out into a separate component? That will increase readability and maintainability as well as it is easier to test each block 😄

Reply:
Moved this code to a utils-function 😄